public/actions.websocket-messages.test.js

```diff
@@ -123,6 +123,33 @@ test('can complete timer from websocket message', t => {
   );
 });
 
+test('can complete a paused timer from websocket message', t => {
+  const initialState = {
+    timerStartedAt: null,
+    timerDuration: 5000,
+  };
+
+  const state = actions.UpdateByWebsocketData(initialState, {
```

Comment:
Let's make that `const [state, fx]`

Reply:
@DigitalEpidemic not sure why you couldn't use `const [state, effect]` here, but I didn't have any issue on my end.

Reply:
@mrozbarry it lets me use it, but when you actually get the test to fail (like by switching the `state.timerDuration === 0` back to `state.timerStartedAt === null` in the `actions.js` file. The failing error will always be:
```
TypeError {
  message: 'undefined is not a function',
}
```
Rather than comparing the actual output vs the expected output when `state.timerStartedAt === null` is used instead of `state.timerDuration === 0`.

Here is the output of a failing test when using `state[0]` and `state[1]`:
![Screen Shot 2020-11-25 at 10 38 09 PM](https://user-images.githubusercontent.com/10665325/100305447-1f6c5200-2f6f-11eb-8baf-66b3c876a53f.png)

Here is when `[state, effect]` is used:
![Screen Shot 2020-11-25 at 10 38 56 PM](https://user-images.githubusercontent.com/10665325/100305473-2eeb9b00-2f6f-11eb-9cb5-250b8b7d3891.png)

If that doesn't matter, then I'm completely okay with switching back over to the `[state, effect]` way! I was just doing this was cause it felt more useful when needing to debug the failing test.